wain-ast/src/lib.rs

```diff
@@ -398,184 +397,184 @@ pub enum InsnKind {
 }
 
 impl InsnKind {
+    #[must_use]
     pub fn name(&self) -> &'static str {
-        use InsnKind::*;
```

Comment:
I'd like to avoid massive repeat of `Self::` here. So previous implementation looks better for me.